templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url "taxi:index" %}" style="color: #f5dd5d">Home page</a>
+  </li>
+  <li>
+    <a href="#" style="color: #f5dd5d">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#" style="color: #f5dd5d">Cars</a>

```

Comment:
The link for 'Cars' is a placeholder. If there is a specific view or URL for this section, update this link to use the `{% url %}` template tag to point to the correct path.